src/routes/dashboard/components/Dashboard.js

```diff
@@ -42,7 +43,7 @@ export type PropsType = {|
 
 class Dashboard extends React.Component<PropsType> {
   getNavigationTileModels (cityCode: string, language: string): Array<TileModel> {
-    // check if news is enabled to show the menu item
+    // Check if news is enabled to show the menu item
```

Comment:
Probably not the right place for this comment

Reply:
Yes, that should've been changed after `dev` is merged. Fixed